packages/filezilla.rb

```diff
@@ -30,6 +30,7 @@ class Filezilla < Package
   depends_on 'wxwidgets'
   depends_on 'xdg_utils'
   depends_on 'sommelier'
+  depends_on 'libwebp'
```

Comment:
Can we put this above sommelier so users don't miss the all-important post install message?

Reply:
done